xbmc/Autorun.cpp

```diff
@@ -308,7 +308,7 @@ bool CAutorun::RunDisc(IDirectory* pDir, const std::string& strDrive, int& nAdde
             
             // Single *.xpl or *.ifo files require an external player to handle playback.
             // If no matching rule was found, VideoPlayer will be default player.
-            if (hdVideoPlayer != "videoplayer")
+            if (hdVideoPlayer != "VideoPlayer")
```

Comment:
this check is imo rather useless. no guarantees that the external player can handle it. we can just drop this comparison.